class/PostTypes/CptBase.php

```diff
@@ -74,6 +74,8 @@ public static function create_rewrite_rules()
     $template_variable = $resource->get_permalink_template_variable();
     $post_permalink_template = static::get_permalink_template();
 
+    if(!$post_permalink_template) return;
```

Comment:
This is a base class used for posts and people FYI - but there's no need for this as the post and person permalink templates aren't allowed to be empty.

Reply:
I have now removed this check